app/controllers/application_controller.rb

```diff
@@ -36,6 +36,10 @@ def server_error
     show_error_page_and_increment_statsd(500)
   end
 
+  def page_title=(title)
```

Comment:
We keep repeating the suffix "MOJ Parliamentary Questions" over and over again.. Changing it later on will be tedious and error prone. How about something like this instead:

    def page_title(title, suffix_override = nil)
        suffix = suffix_override || DEFAULT_TITLE_SUFFIX
       @page_title = [title, suffix].compact.join(' - ')
    end